### PR TITLE
Feature/createuser

### LINKED ISSUE #34: External User Sign Up
Create a feature that allows external user to sign up. On sign up  a checking account should be created automatically.

### LINKED ISSUE #35: Internal User Sign Up
Add a feature to allow admin to add new user to internal system.
- Admin should enter email and role.
- An email should be sent to user
- User can use link provide to sign up with the given email address

### PR DESCRIPTION
**Pull request by ayush2k**
Resolves #35, Resolves #34 .
Description:
- External User sign up
- Internal User sign up
- Email Service
- Validators
- Details Page
- All Users Page

### CODE REVIEW

**Comment by ayush2k** [comment]
@joelmascarenhas @shivanijjwala @madhuilluri @Mitikaa please review

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,14 +1,29 @@
-spring.datasource.driver-class-name = com.mysql.jdbc.Driver
-spring.datasource.url: jdbc:mysql://localhost:3306/securbank?useSSL=false
-spring.datasource.username: root
+/pring.datasource.driver-class-name = com.mysql.jdbc.Driver
```
Comment (wchrismontgomery) [inline src/main/resources/application.properties]: This typo should be fixed.

Comment (wchrismonty) [inline src/main/resources/application.properties]: This typo should be fixed.

**Review by wchrismonty — CHANGES_REQUESTED** [review]

File: src/main/resources/application.properties (hunk 1 above)
Comment (ayush2k) [reply src/main/resources/application.properties]: @wchrismontgomery fixed. thanks!

**Review by wchrismonty — APPROVED** [review]

**Pull request merged**